tutorials/colors/colormaps.py

```diff
@@ -221,8 +223,8 @@ def plot_color_gradients(cmap_category, cmap_list, nrows):
         ax.set_axis_off()
 
 
-for cmap_category, cmap_list in cmaps.items():
-    plot_color_gradients(cmap_category, cmap_list, nrows)
+for i, (cmap_category, cmap_list) in enumerate(cmaps.items()):
+    plot_color_gradients(cmap_category, cmap_list, nrows[i])
```

Comment:
```suggestion
for rows, (cmap_category, cmap_list) in zip(nrows, cmaps.items()):
    plot_color_gradients(cmap_category, cmap_list, rows)
```
but do we really need `nrows` now? Can we not just check the length of `cmap_list` in `plot_color_gradients`?